runtime/src/lib.rs

```diff
@@ -295,6 +335,114 @@ impl pallet_sudo::Config for Runtime {
 impl pallet_chocolate::Config for Runtime {
 	type Event = Event;
 }
+/// Configure the pallet-users in pallets/users.
+impl pallet_users::Config for Runtime {
```

Comment:
Shoudn't we need to add something more here?

Reply:
Not necessarily, the types defined in the chocolate module are simply aliases,  hence the y don't need any external configuration.